sos/report/__init__.py

```diff
@@ -1362,11 +1357,8 @@ def collect_plugin(self, plugin):
         except Exception:
             return False
         numplugs = len(self.loaded_plugins)
-        status_line = "  Starting %-5s %-15s %s" % (
-            "%d/%d" % (count, numplugs),
-            plugname,
-            "[Running: %s]" % ' '.join(p for p in self.running_plugs)
-        )
+        status_line = (f"  Starting {count}/{numplugs:<5} {plugname:<15} "
```

Comment:
Same like above. Now we limit the size of whole `{count}/{numplugs}`, newly we would limit just `{numplugs}`.